packages/camera/camera/ios/Classes/FlashMode.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <AVFoundation/AVFoundation.h>
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Represents camera's flash mode. Mirrors `FlashMode` enum in flash_mode.dart.
+ */
+typedef enum {
+  FlashModeOff,
+  FlashModeAuto,
+  FlashModeAlways,
+  FlashModeTorch,
+} FlashMode;
```

Comment:
The Obj-C faux "name spacing" convention is a different three letter prefix per module.  So each plugin should have a different prefix, which should be different from Flutter's which is `FTL`.  But unfortunately they don't and it's not consistent. https://google.github.io/styleguide/objcguide.html#prefixes

So these symbols need some kind of prefix like `FTLFlashMode`, and the enum values should extend the typedef name for Swift compatibility.  Also use `NS_ENUM`. https://google.github.io/styleguide/objcguide.html#constants
```obj-c
typedef NS_ENUM(NSInteger, FTLFlashMode) {
  FTLFlashModeOff = 1,
  FTLFlashModeAuto = 2,
  FTLFlashModeAlways = 3,
  FTLFlashModeTorch = 4,
};
```

Reply:
Isn't it `FLT`?

But we could start using a different prefix for each plugin (or at least a different prefix for this repo vs others, like `FLTP`; as you said there's not much consistency right now (including Flutter itself not using a prefix in many cases, which drives me crazy) so we could just start incrementally improving.

Reply:
will go with `FLTFlashMode`

Reply:
Actually, should we go with `CAM` for camera?

Reply:
@jmagman Do you just want to pick a prefixing policy for this repo that you're happy with that we can enforce going forward? (We can add a quick note about it in the ObjC section [here](https://github.com/flutter/plugins/blob/main/CONTRIBUTING.md#style)).

Reply:
updated to `FLT` to be consistent with other types in camera for now, but happy to change to anything else

Reply:
I guess stick with `FLT` for now, I like `CAM` better but we can change it in another PR, and I can update the obj-c section as Stuart suggested.

> Isn't it `FLT`?

You're right it's not
<img src="https://user-images.githubusercontent.com/682784/151246805-f83b006a-fb2a-4764-aee2-d9672fb88847.jpg" width=400">